src/table/TableBodyWrapper.jsx

```diff
@@ -67,7 +67,7 @@ const TableBodyWrapper = ({
   }, []);
 
   return (
-    <TableBody className={classes.cellBase}>
+    <TableBody className={classes.cellBase} style={{ height: `${rootElement.clientHeight - 102}px` }}>
```

Comment:
Why 102? Would be nice if you can get this in a class, but not in `cellBase` since it is not related to cell.

Reply:
the height of <thead> + the height of <TablePagination>, which are fixed value. Sure.

Reply:
the height of what and what? 🙃

Reply:
😂 sorry, I should preview it first. the height of` <TableHead>` + the height of` <TablePagination>`

Reply:
OK, no problem. Well the height of the TableHead varies with the font size?

Reply:
good point. I need to update accordingly.